runtime/compiler/optimizer/DataAccessAccelerator.cpp

```diff
@@ -94,7 +94,13 @@ int32_t TR_DataAccessAccelerator::perform()
        !comp()->getOption(TR_MimicInterpreterFrameShape) &&
 
        // We cannot handle arraylets because hardware intrinsics act on contiguous memory
-       !comp()->generateArraylets()&& !TR::Compiler->om.useHybridArraylets())
+       !comp()->generateArraylets()&& !TR::Compiler->om.useHybridArraylets() &&
```

Comment:
```suggestion
       !comp()->generateArraylets() && !TR::Compiler->om.useHybridArraylets() &&
```

I understand it existed already, but a nitpick suggestion.